### ISSUE TITLE
Request: Support retroachievements

### ISSUE THREAD

**Issue by DukeJP** [punesemu/puNES#49]
Please, add support to the retroachievements.org page. 
Would be a nice feature.
You can do this in two ways: 1.-add support to the puNES emulator directly, or 2.- Make a Libretro port of puNES, Libretro already has support for retroachievements.
The other nes cores in Libretro arent too good. puNES is close to be the best nes emulator, would be great use this emu for accurate emulation.

**Comment by slacknk** [comment]
Good idea. If add, then puNES - 1st linux qt-win emulator this [feature](https://docs.retroachievements.org/FAQ/)
